### ISSUE TITLE
Sticky ad whenBuilt is not a function

### ISSUE THREAD

**Issue by muxin** [ampproject/amphtml#9123]
```
TypeError: this.b.whenBuilt is not a function
    at z.push.f.g.buildCallback (https://cdn.ampproject.org/rtv/001493770802531/v0/amp-sticky-ad-1.0.js:2:290)
    at amp-sticky-ad.Tf.b.build (https://cdn.ampproject.org/rtv/001493770802531/v0.js:149:514)
    at ch.f.build (https://cdn.ampproject.org/rtv/001493770802531/v0.js:231:151)
    at qh (https://cdn.ampproject.org/rtv/001493770802531/v0.js:251:355)
    at ih.f.upgraded (https://cdn.ampproject.org/rtv/001493770802531/v0.js:253:170)
    at amp-sticky-ad.Tf.b.qc (https://cdn.ampproject.org/rtv/001493770802531/v0.js:148:328)
    at amp-sticky-ad.Tf.b.Ge (https://cdn.ampproject.org/rtv/001493770802531/v0.js:160:447)
    at amp-sticky-ad.Tf.b.connectedCallback (https://cdn.ampproject.org/rtv/001493770802531/v0.js:159:381)
```

**Comment by muxin** [comment]
Related PR https://github.com/ampproject/amphtml/pull/8815

**Comment by zhouyx** [comment]
Working on a fix.

**Comment by zhouyx** [comment]
I merged #9127 
@muxin Please cherry pick. Thanks!
